### PR TITLE
Testing/robot

### LINKED ISSUE #107: E2E: automated testing with robot
- [ ] poetry init
- [ ] poetry add <robot deps>
- [ ] create a simple sign in E2E robot test
- [ ] raise PR

### PR DESCRIPTION
**Pull request by RedValis**
### Description

Resolves #107 

To be reviewed by: @Nasfame

### CODE REVIEW

File: tests/Test1.robot [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Test Head
+    Log To Console    test_head
+    Open Browser    ${URL}    ${BROWSER}
+    Sleep    5
+    Page Should Contain  DecentralIzed and DemocratIzed AI 
+    Page Should Contain  Model TraInIng Infrastructure
+    Sleep    2
+
+
+    Page Should Contain Button  Try Demo
```
Comment (Nasfame) [inline tests/Test1.robot:22]: can u raise a pr for the same change over DecenterAI/decenterai.com. 

This is the repo for app.decenterai.com. And pleace replace the url with localhost:3000

**Comment by Nasfame** [comment]
@RedValis update ur branch

File: tests/Test1.robot (hunk 1 above)
Comment (RedValis) [reply tests/Test1.robot:22]: alr, got it

Comment (Nasfame) [reply tests/Test1.robot:22]: then do it

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "app-decenterai-com"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+packages = [{include = "app"}]
+
+[tool.poetry.dependencies]
+python = "^3.10"
```
Comment (Nasfame) [inline pyproject.toml:10]: poetry add robot framework. Refer decenterai-streamlit.app @RedValis

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "app-decenterai-com"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+packages = [{include = "app"}]
```
Comment (Nasfame) [inline pyproject.toml:7]: we don't have an app package remove this

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 3 above)
Comment (Nasfame) [reply pyproject.toml:7]: @RedValis i do see it now

File: pyproject.toml (hunk 2 above)
Comment (Nasfame) [reply pyproject.toml:10]: good job

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 3 above)
Comment (Nasfame) [reply pyproject.toml:7]: kindly remove {include="app"} and leave it as []

Comment (RedValis) [reply pyproject.toml:7]: Sure got it

Comment (RedValis) [reply pyproject.toml]: Done

**Pull request closed without merging**